lib/spatial_features/importers/kml.rb

```diff
@@ -35,7 +35,7 @@ def geom_from_kml(kml)
 
         # Do query in a new thread so we use a new connection (if the query fails it will poison the transaction of the current connection)
         Thread.new do
-          geom = ActiveRecord::Base.connection.select_value("SELECT ST_GeomFromKML(#{ActiveRecord::Base.connection.quote(kml.to_s)})")
+          geom = SpatialFeatures::Utils.select_db_value("SELECT ST_GeomFromKML(#{ActiveRecord::Base.connection.quote(kml.to_s)})")
         rescue ActiveRecord::StatementInvalid => e # Discard Invalid KML features
```

Comment:
This in particular feels weird.  We're calling into a utility function but then rescuing from ActiveRecord, so we're not really encapsulating anything.  Any thoughts?